testsuites/CBLTester/CBL_Functional_tests/APITests/test_collection.py

```diff
@@ -0,0 +1,264 @@
+import pytest
```

Comment:
In general, a test should cleanup after itself. So, for example, if it adds a collection, it should delete it at teardown. 
Since we're creating everything from scratch for every test it might not be relevant, but we do want to move away from that at a certain point (I think...), and tests that don't clean up after themselves would be an issue then, so it might be worth starting to start cleaning up new tests now.